package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "es6-shim": "0.35.8",
-    "express": "^4.19.2",
+    "express": "^4.21.0",
```

Comment:
Express 4.21.1 was released on 2024-10-08 to address an issue with a vulnerability on the `cookie` npm
https://github.com/expressjs/express/issues/6026